pom.xml

```diff
@@ -1869,6 +1869,25 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-tools</artifactId>
```

Comment:
Sure, this is only here temporarily pending ARROW-411. Feel free to cherry pick this commit and modify to suit